IronPython.sln

```diff
@@ -295,13 +297,39 @@ Global
 		{79446DDF-421C-4695-9859-5E01F0D95830}.Win8Debug|Any CPU.Build.0 = Win8Debug|Any CPU
 		{79446DDF-421C-4695-9859-5E01F0D95830}.Win8Release|Any CPU.ActiveCfg = Win8Release|Any CPU
 		{79446DDF-421C-4695-9859-5E01F0D95830}.Win8Release|Any CPU.Build.0 = Win8Release|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.AndroidDebug|Any CPU.ActiveCfg = AndroidDebug|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.AndroidDebug|Any CPU.Build.0 = AndroidDebug|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.AndroidRelease|Any CPU.ActiveCfg = AndroidRelease|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.AndroidRelease|Any CPU.Build.0 = AndroidRelease|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.FxCop|Any CPU.ActiveCfg = FxCop|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.FxCop|Any CPU.Build.0 = FxCop|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.Release|Any CPU.Build.0 = Release|Any CPU
+		{3DFB096E-AC09-4E7A-9288-7F7C33C288C7}.Silverlight5Debug|Any CPU.ActiveCfg = Silverlight5Debug|Any CPU
```

Comment:
Yes, it will, but this merge was not about removing anything, it was only bringing ironpython3 up to par with main. I still need to do one of these merges for the DLR.